.github/workflows/deploy-prodbox.yml

```diff
@@ -18,10 +27,13 @@ concurrency:
 
 env:
   GCLOUD_PROJECT_ID: ${{ secrets.GCLOUD_PROJECT_ID }}
+  IMAGE_NAME: prodbox
 
 jobs:
-  build-and-deploy:
+  notify-start:
```

Comment:
ooc what does this do?

Reply:
This opens a slack thread on #deployments 😃